convert.py

```diff
@@ -0,0 +1,108 @@
+from xml.etree import ElementTree as ET
+import json
+import sys
+
+orig_stdout = sys.stdout
+f = open('output.txt', 'w')
+sys.stdout = f
+
+with open("batch_response_data.json") as json_file:
+    json_data = json.load(json_file)
+
+
+ns = {'ns': 'http://www.transunion.com/namespace/pfs/v4'}
+for data in json_data:
+    root = ET.fromstring(data)
+        
+    print "TRANSUNION CREDIT REPORT\n"
+    print "<FOR>           <SUB NAME>          <MKT SUB>  <INFILE>    <DATE>     <TIME>"
+    print "(I) " + ("X" * len(root.find(".//ns:industryCode", ns).text)).ljust(12) + "XXXXX XXXXX".ljust(20) + "00 XX".ljust(11) + "0/00".ljust(12) + "00/00/00".ljust(11) + "10:00CT\n"
+    print "<SUBJECT>                                            <SSN>       <BIRTH DATE>"
+    print ("X" * len(root.find('.//ns:last', ns).text) + ", " + "X" * len(root.find('.//ns:first', ns).text)).ljust(53) + "XXX".ljust(12) + "00/00/00".ljust(12)
+    print "<CURRENT ADDRESS>                                                 <DATE RPTD> "
+    indicative = root.findall('.//ns:indicative', ns)[1]
+    print ("X" * len(str(indicative[1][0][0].text)) + ", " + "X" * len(str(indicative[1][1][0].text)) + " " + "X" * len(str(indicative[1][1][1].text)) + ". " + "0" * len(str(indicative[1][1][2].text))).ljust(66) 
+    print "<FORMER ADDRESS>"
+    former_address = indicative[2]
+    if former_address.find('.//ns:address', ns) is None:
+        print ""
+    else:
+        print ("X" * len(str(indicative[2][0][1][0].text)) + ", " + "X" * len(str(indicative[2][0][2][0].text)) + " " + "X" * len(str(indicative[2][0][2][1].text)) + ". " + "0" * len(str(indicative[2][0][2][2].text))).ljust(66)
+    print "                                        <POSITION>"
+    print "<CURRENT EMPLOYER AND ADDRESS                           <RPTD><HIRE>"
+    print "XXXX".ljust(40) + "XX".ljust(16) + "0/00".ljust(4) + "0/00".ljust(4)
+    print "<FORMER EMPLOYER AND ADDRESS>"
+    print "XXXX".ljust(40) + "XX".ljust(16) + "0/00".ljust(4) + "0/00".ljust(4)
+    print ""
+    print "                  ***WARNING INTERNAL USE ONLY - NOT FOR RESALE***"
+    print "--------------------------------------------------------------------------------------"
+    print "S P E C I A L M E S S A G E S"
+    print "****HIGH RISK FRAUD ALERT:INPUT SSN NOT ISSUED BY SOCIAL SECURITY"
```

Comment:
alerts and warnings can be found in the xml in the determinationStatus element. Please use those to fill in this section